.github/commit-cop.yml

```
@@ -0,0 +1,4 @@
+requireTitle: true
+requireCommits: 'all'
```

if they plan on just doing squashes and not merged

```suggestion
requireCommits: 'some'
```

---

@sjungling merge commit this branch please